### DISCUSSION TITLE
javamusic bot problem

### DISCUSSION THREAD

**Discussion by silenta00** [jagrosh/MusicBot#1707] [Troubleshooting] [answered]
i did everything as it said , i used to play the bot perfectly in the previous versions and now when it asked me to download the new one is not working , i deleted java and re downloaded it but still same problem

![Screenshot_8](https://github.com/user-attachments/assets/9f74e83f-08fb-4822-998e-71dd12047385)
![Screenshot_16](https://github.com/user-attachments/assets/7cb46e3f-a44a-4566-a779-d0a99191e1c6)

just for more information after i re-downloaded Java only the second picture appears

**Comment by MichailiK** [comment]
You installed Java 8. You need to install Java 11 or later to use JMusicBot.

**Reply by silenta00** [reply]
thank you so much for that now i have another problem and i don't know how to disable it , is there any link to follow instructions?

![Screenshot_20](https://github.com/user-attachments/assets/a41d059b-10fd-4623-8ff1-5813a9dea54c)

**Reply by MichailiK** [reply]
The error message tells you to go to https://discord.com/developers/applications/628569463893000202/bot and disable public bot. It also tells you that you may need to go to https://discord.com/developers/applications/628569463893000202/installation to disable the installation contexts ("Guild install" and "User install" checkboxes).

**Reply by silenta00** [reply]
i did everything as you said , i'm really sorry for being annoying and i asked a lot of questions 

when i play the music the bot join the room and then leave immediately idk what's the problem

**Reply by MichailiK** [reply]
YouTube has recently changed stuff that broke JMusicBot for some people. You might be experiencing one of these two issues:

- #1588
- #1682

A fix for both of these issues will be released in the next version of JMusicBot. Until then, you can use other services like SoundCloud, or play local files.

**Reply by silenta00** [reply]
this is what it said to me , so i should just wait for the next version right?

![Screenshot_21](https://github.com/user-attachments/assets/2322036d-1808-4cc0-b923-ed084798cf1f)

**Reply by MichailiK** [reply, marked as answer]
You're experiencing #1682 so yes, correct. In the meantime though you can use sources other than YouTube, e.g. Soundcloud or local files
